src/NuGet.Clients/NuGet.PackageManagement.UI/Converters/BooleanToWidthConverter.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Data;
+
+namespace NuGet.PackageManagement.UI
+{
+    internal class BooleanToWidthConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            var boolValue = (bool)value;
+
+            return boolValue ? "Auto" : "0";
```

Comment:
Have you had a chance to test the narrator here? 
https://github.com/NuGet/NuGet.Client/blob/dev/docs/feature-guide.md#visual-studio-ui-considerations

Reply:
I completely refactored this and Narrator indicates correct # columns now.